tests/unit/test_docs.py

```diff
@@ -0,0 +1,43 @@
+"""Test for the sphinx documentation."""
+import subprocess
+import glob
+import os
+import pytest
+
+UNDOCUMENTED_FILES = ["__init__", "constants", "lib_mapper", "protocol_mapper", "variables"]
+
+SPHINX_DIRECTORIES = [
+    {
+        "source_dir": "docs/source/",
+        "build_dir": "docs/build/",
+    }
+]
+
```

Comment:
Can you add a start/end line test here? Something to ensure that the current start and end line remain the same throughout updates. This will ensure that if the readme is modified, that the tests will fail. If the start or end line change, they simply need to modify the tests.

Reply:
Added tests to check for that. Let me know what you think.